### PR TITLE
Improvement/file structure

### PR DESCRIPTION
**Pull request by ghost**
## Description about the changes

Updated the file structure:
- created subfolder for `evaluator`, `parser`,  and `utils` (errors, baseclases, etc)

## Python Version you tested this feature on

- [ ] Python 3.7
- [ ] Python 3.8
- [ ] Python 3.9
- [X] Python 3.10
- [ ] Python 3.11

### CODE REVIEW

File: datetimeparser/parser/parser.py [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+from datetimeparser.parser.parsermethods import (
     AbsoluteDateFormatsParser,
-    RelativeDatetimesParser,
-    ConstantsParser,
+    AbsolutePrepositionParser,
     ConstantRelativeExtensionsParser,
+    ConstantsParser,
     DatetimeDeltaConstantsParser,
-    AbsolutePrepositionParser
+    RelativeDatetimesParser
```
Comment (Ari24-cb24) [inline datetimeparser/parser/parser.py:9]: Why the change of import order here?

**Review by Ari24-cb24 — APPROVED** [review]
Let's hope for the best. Also, might need to change the imports in the testrunner as they append a dot dot (directory parent) to the module path whenever the imports fail

File: datetimeparser/parser/parser.py (hunk 1 above)
Comment (ghost) [reply datetimeparser/parser/parser.py:9]: It triggered my sense for aesthetic -> had to sort it in alphabetical order
